clients/extension/src/ExtensionClient.ts

```diff
@@ -66,7 +66,7 @@ export class ExtensionClient extends Client {
     const options: RequestOptions = { method: "GET" };
     await this.setupOptionDefaults(options);
     requestContext.enter();
-    options.headers = { authorization: requestContext.accessToken.toTokenString() };
+    options.headers = { authorization: requestContext.accessToken };

```

Comment:
I believe Caleb mentioned retaining only `Bearer` tokens. Is this still what you are thinking @calebmshafer ?

Reply:
I've added a function to `Token.ts` called `removeAccessTokenPrefix` that does the same thing as the `.substr` function so that it can be reused more efficiently.